service/src/main/kotlin/app/cash/backfila/dashboard/SeedDbAction.kt

```diff
@@ -0,0 +1,77 @@
+package app.cash.backfila.dashboard
+
+import app.cash.backfila.api.ConfigureServiceAction
+import app.cash.backfila.api.CreateAndStartBackfillAction
+import app.cash.backfila.protos.service.ConfigureServiceRequest
+import app.cash.backfila.protos.service.CreateAndStartBackfillRequest
+import app.cash.backfila.protos.service.CreateBackfillRequest
+import javax.inject.Inject
+import misk.MiskCaller
+import misk.inject.keyOf
+import misk.scope.ActionScope
+import misk.web.Get
+import misk.web.ResponseContentType
+import misk.web.actions.WebAction
+import misk.web.mediatype.MediaTypes
+
+class SeedDbAction @Inject constructor(
+  val configureServiceAction: ConfigureServiceAction,
+  val createAndStartBackfillAction: CreateAndStartBackfillAction,
+  val scope: ActionScope,
+) : WebAction {
+
+  @Get("/seed")
```

Comment:
Rather than an endpoint don't we want a startup action?

Reply:
I think that's actually the right way to do it, but didn't know offhand how/where to hook that in.